packages/core/src/browser/tree/tree-container.spec.ts

```diff
@@ -0,0 +1,28 @@
+// *****************************************************************************
+// Copyright (C) 2022 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+// *****************************************************************************
+import * as assert from 'assert';
+import { isTreeServices } from './tree-container';
+import { TreeSearch } from './tree-search';
+import { defaultTreeProps } from './tree-widget';
+
+describe('IsTreeServices should accurately distinguish TreeProps from TreeContainerProps', () => {
```

Comment:
My only suggestion would be to have a top level `describe` to describe the whole file before we have individual test-cases:

```ts
describe('tree-container', () => {
...
});
```

It helps with the output of the tests, and helps group additional tests later for the same file.